lib/pytis/remote/clientapi.py

```
@@ -1049,6 +1042,25 @@ def _open_file(self, filename, mode, encoding, encrypt=None, decrypt=False):
+        else:
+            os_version = platform.release()
+        # Note: We are serializing the dict to prevent RPyC sending it as a netref,
+        # which seems to be the default with older RPyC versions (3.4.4).
+        return json.dumps(dict(
+            os_name=os_name,
+            os_version=os_version,
+            backend_name=self._client.name(),
+            python_version=sys.version_info[:3],
+            # Note: The rpyc.__version__ contents is inconsistent across rpyc versions.
+            rpyc_version=(rpyc.__version__ if isinstance(rpyc.__version__, tuple) else
+                          tuple(map(int, rpyc.__version__.split('.')))),
```

Ruff (0.12.2)

`__version__` may contain non-integral-like elements

(RUF048)